packages/v4/src/content/design-guidelines/styles/typography/typography.js

```diff
@@ -21,73 +21,99 @@ export const TitleLevel = ({ asGrid, styleProps, children, title = '', note = ''
 }
 
 export const styleProps = {
+  superHero: {
+    fontWeight: "400",
+    fontWeightText: "400 (medium)",
+    variableName: "--pf-global--FontSize--4xl",
+    fontSize: "36px",
+    lineHeight: "1.3",
+    fontFamily: "RedHatDisplay",
+    textStyle: "4xl (RHD md 36)"
+  },
+  hero: {
+    fontWeight: "400",
+    fontWeightText: "400 (medium)",
+    variableName: "--pf-global--FontSize--3xl",
+    fontSize: "28px",
+    lineHeight: "1.3",
+    fontFamily: "RedHatDisplay",
+    textStyle: "3xl (RHD md 28)"
+  },
   first: {
     fontWeight: "400",
     fontWeightText: "400 (medium)",
     variableName: "--pf-global--FontSize--2xl",
     fontSize: "24px",
     lineHeight: "1.3",
-    fontFamily: "RedHatDisplay"
+    fontFamily: "RedHatDisplay",
+    textStyle: "2xl (RHD md 24)"
   },
   second: {
     fontWeight: "400",
     fontWeightText: "400 (medium)",
     fontSize: "20px",
     variableName: "--pf-global--FontSize--xl",
-    lineHeight: "1.3",
-    fontFamily: "RedHatDisplay"
+    lineHeight: "1.5",
+    fontFamily: "RedHatDisplay",
+    textStyle: "xl (RHD md 20)"
   },
   third: {
     fontWeight: "400",
     fontWeightText: "400 (medium)",
     fontSize: "18px",
     variableName: "--pf-global--FontSize--lg",
     lineHeight: "1.5",
-    fontFamily: "RedHatDisplay"
+    fontFamily: "RedHatDisplay",
+    textStyle: "lg (RHD md 18)"
   },
   fourth: {
     fontWeight: "400",
     fontWeightText: "400 (medium)",
     fontSize: "16px",
     variableName: "--pf-global--FontSize--md",
     lineHeight: "1.5",
-    fontFamily: "RedHatDisplay"
+    fontFamily: "RedHatDisplay",
+    textStyle: "Md (RHD md 16)"
```

Comment:
noticed you wrote "lg" lower case, but here and in Sm you wrote them with capital M and S. Would make sure they're all consistent and probably go with all lower case? (ie: md, sm)